src/IntroScreen.h

```diff
@@ -13,7 +13,7 @@ using TextList = std::vector<TextPtr>;
 class IntroScreen : public Screen
 {
     sf::Font        _font;
-    sf::Texture     _bgt;
+    sf::Texture     _bgt[8];
```

Comment:
Here it would be better to use `std::vector`. You could declare it like so:
```
std::vector<sf::Texture> _bgt;
```

Then when you're putting things into it you could do:
```
const auto filename = fmt::format("images/ttvg-intro-screen-{}.png", i+1);
auto bgt = _resources.load<sf::Texture>(filename);

// check to make sure it's loaded

_bgt.push_back(std::move(bgt));
```

This will avoid the copy you ask about above since it's using move-semantics. Note that after you use `std::move` that the value of `bgt` itself will no loner be valid. 

And lastly, I would recomend using `_bgt.at(i)` to access the items in the vector, as opposed to `_bgt[i]`, as long as you don't need to modify the element. `at()` gives you const-access to the element, which is generally preferable.